weed/server/filer_server_handlers_read.go

```diff
@@ -131,6 +131,9 @@ func (fs *FilerServer) GetOrHeadHandler(w http.ResponseWriter, r *http.Request,
 
 	if r.Method == "HEAD" {
 		w.Header().Set("Content-Length", strconv.FormatInt(totalSize, 10))
+		processRangeRequest(r, w, totalSize, mimeType, func(writer io.Writer, offset int64, size int64) error {
```

Comment:
The clients sending HEAD requests not necessarily want to check the chunks.  How to get around this?

Reply:
> The clients sending HEAD requests not necessarily want to check the chunks. How to get around this?

I don't understand, why then send a HEAD request? and why run out of it?
Since this is the only mechanism to understand before uploading a file in a stream, is everything good with it?

Reply:
Here you need to download a 100 gig file and 10500 chunks, and here it was loaded for 10 hours, and on 10499 chunk an error occurred due to the absence of a chunk, and at the same time a HEAD request that checks for the presence of only the first chunk works, why is such a check from a single chunk needed?

Reply:
HEAD request may just want to get the file size, or modified time.

Reply:
> HEAD request may just want to get the file size, or modified time.

Yes, this is often needed for synchronization and it is better to check the liveness and return an error on HEAD than to return 200 and a broken file during synchronization

Reply:
checking file health is different from checking file size. The file chunks may be offline for some reason. But it should not fail the HEAD opertion.

Reply:
> checking file health is different from checking file size. The file chunks may be offline for some reason. But it should not fail the HEAD operation.

Well i realized